### PR TITLE
Basic Image Upload & Unique Room Creation

### PR DESCRIPTION
**Pull request by Aneesh02**

### CODE REVIEW

**Comment by Aneesh02** [comment]
@KShivendu Help required to merge RegisterUser and RegisterUser2 in views.py

**Review by KShivendu — COMMENTED** [review]
Add a `.gitignore` file and remove the images, .sqlite3 and other unnecessary files.

File: backend/main/home/admin.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from django.contrib import admin
+from .models import Photo
+
+# Register your models here.
+admin.site.register(Photo)
```
Comment (KShivendu) [inline backend/main/home/admin.py:5]: Add an empty line here. (This is a common convention to have an empty line at the end of the files)

Comment (KShivendu) [reply backend/main/home/admin.py:5]: [Reason](https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file)

File: backend/main/home/models.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from django.contrib.auth.models import User
+# Create your models here.
+# class Room(models.Model):
+#     code = models.CharField(max_length=10)
+#     pwd = models.CharField(max_length=122)
+
+#     def __str__(self):
+#         return self.code
+
```
Comment (KShivendu) [inline backend/main/home/models.py:10]: Always try to remove unused code. It makes the file ugly.

File: backend/main/home/models.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+class Photo(models.Model):    
+    user = models.ForeignKey(User,on_delete=models.CASCADE)
+    image = models.ImageField(null=False, blank=False)
+    def __str__(self):
+        return self.user
```
Comment (KShivendu) [inline backend/main/home/models.py:16]: Add empty newline here

File: backend/main/home/urls.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+urlpatterns = [
+    path('',views.index,name="index"),
+    path('login', views.loginUser, name="login"),
+    path('logout', views.logoutUser, name="logout"),
+    path('register', views.registerUser, name="register"),
+    path('register2', views.registerUser2, name="register2"),
+    path('photo/<str:pk>/', views.viewPhoto, name='photo'),
+    path('add',views.addPhoto,name="add"),
+    path('process',views.process,name="process"),
+]
```
Comment (KShivendu) [inline backend/main/home/urls.py:13]: Add Newline.

You should be able to run black formatter on the whole repo. It will fix a lot of such mistakes.

File: backend/main/main/settings.py [hunk 5]
```diff
@@ -49,12 +50,12 @@
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 ]
 
-ROOT_URLCONF = 'core.urls'
+ROOT_URLCONF = 'main.urls'
```
Comment (KShivendu) [inline backend/main/main/settings.py:53]: Why did you delete the `core` app and create a new project called `main`? 

Please rename it to main. (You might have to recreate the project)

File: backend/main/main/settings.py [hunk 6]
```diff
@@ -101,25 +102,34 @@
...
+#added manually
+STATICFILES_DIRS = [
+    BASE_DIR / "static",
+]
+
+
+
+
+MEDIA_ROOT = BASE_DIR / 'static/images'
+STATIC_ROOT = BASE_DIR / 'staticfiles'
```
Comment (KShivendu) [inline backend/main/main/settings.py:135]: Add newline (or run black on the whole repo as previously suggested)

**Review by KShivendu — COMMENTED** [review]
Add `.pyc` files to `.gitignore`

**Comment by Aneesh02** [comment]
Incorporated all your changes. Check final push: https://github.com/OpenLake/Photo-Sharing-App/pull/6
